schema.graphql

```diff
@@ -80,14 +98,39 @@ type Loan @entity {
   id: ID!
   transaction: EthTransaction!
   borrower: Borrower!
+  recipient: Bytes!
+  terms: LoanTerm!
   startDate: BigInt! # https://github.com/graphql/graphql-js/issues/1003#issuecomment-324790647
+  endDate: BigInt!
+  amountBorrowed: BigInt!
+  status: LoanStatus!
+  repayments: [LoanRepayment!]!
+  totalRepaidAmount: BigInt!
+  collateralDeposits: [CollateralDeposit!]!
+  totalCollateralDepositsAmount: BigInt!
+  collateralWithdrawns: [CollateralWithdraw!]!
+  totalCollateralWithdrawalsAmount: BigInt!
+  blockNumber: BigInt!
+  timestamp: BigInt!
+}
+
+type LoanTerm @entity {
+  id: ID! # LoanID
+  transaction: EthTransaction!
   interestRate: BigInt!
   collateralRatio: BigInt! 
   maxLoanAmount: BigInt!
-  numberDays: BigInt!
-  amountBorrow: BigInt!
-  collateralDeposits: [CollateralDeposit!]!
-  collateralWithdrawns: [CollateralWithdraw!]!
+  duration: BigInt!
+  expiryAt: BigInt!
+  blockNumber: BigInt!
+  timestamp: BigInt!
+}
+
+type LoanRepayment @entity {
+  id: ID!
+  transaction: EthTransaction!
+  amount: BigInt!
+  payer: Bytes!
```

Comment:
@salazarguille who the payer would be when a repayment is made? (the borrower or someone else?)